index.md

```diff
@@ -118,7 +118,7 @@ Along with bug fixes and general improvements, Anbox Cloud 1.13 comes with:
 | 2 | ref/ams-http-api | [AMS HTTP API](https://discourse.ubuntu.com/t/ams-rest-api-reference/17801)|
 | 2 | ref/anbox-https-api | [Anbox HTTP API](https://discourse.ubuntu.com/t/anbox-http-api-reference/17819)|
 | 2 | ref/anbox-stream-gateway | [Stream Gateway API](https://anbox-cloud.github.io/1.10/anbox-stream-gateway/index.html)|
```

Comment:
Do we have a new version of this one as well?

Reply:
Not yet, the docs are currently not building and we have to fix that first.